packages/widget/src/hooks/useDebouncedWatch.ts

```diff
@@ -1,10 +1,8 @@
 import { useEffect, useRef, useState } from 'react';
-import { useWatch } from 'react-hook-form';
+import { useFieldValues } from '../stores';
 
 export const useDebouncedWatch = (name: any, delay: number) => {
```

Comment:
Can we remove `any` and type `name` properly here now?

Reply:
Actually yes and I think the use of the useDebouncedWatch probably doesn't need to be used with FormKey anymore